### DISCUSSION TITLE
Next steps

### DISCUSSION THREAD

**Discussion by silverlayer** [RAWArduino/SSDisplay#3] [General]
Ok Peter, I ran your tests few hours ago. Apparently, it's all right.
You could prepare and compile the docs, whereas I write examples and required files.

**Comment by peterwaka** [comment]
I have written more two examples and generated the documentation. Feel free to improve something and when you finish, please, notify me for I do the merge.

**Reply by silverlayer** [reply]
I wanna do improvements in "readme.md" file, but I think that should be better do it directly in master branch. So, you can merge with no problem.

**Comment by silverlayer** [comment]
Peter, I have finished the enhancement on docs. Now, you can point the url "peterwaka.github.io/ssdisplay" to master branch.
